src/Testing/MockStubTrait.php

```diff
@@ -72,7 +73,9 @@ public function _simpleRequest(
             throw new UnderflowException("ran out of responses");
         }
         list($response, $status) = array_shift($this->responses);
-        return new MockUnaryCall($response, $deserialize, $status);
+        $call = new MockUnaryCall($response, $deserialize, $status);
+        $this->callObjects[] = $call;
```

Comment:
It appends `$call` to the end of the array (it is preferred over array_push: http://php.net/manual/en/function.array-push.php)